src/model.py

```diff
@@ -906,6 +908,8 @@ def get_taint(self) -> InputTaint:
                 # we taint the 64-bits block that contains the address
                 input_offset = (int(label, 16)) // 8
             else:
+                # uncomment if to create violations of vspec-ops-div
+                # if not label == 'D':
```

Comment:
To create a violation according to vspec-ops-div-faulty (to show we need to add all the operands as dependencies), I needed to compromise the input taint tracking so that actual violations are found.